ee/dtr/admin/install/index.md

```diff
@@ -54,99 +58,68 @@ your profile on the left navigation pane, and select **Admin Settings > Docker T
 
     ![](../../images/create-repository-1.png){: .with-border}
 
 
-## Step 5. Configure DTR
-
-After installing DTR, you should configure:
+## 5. Configure DTR
 
-  * The certificates used for TLS communication. [Learn more](../configure/use-your-own-tls-certificates.md).
-  * The storage backend to store the Docker images. [Learn more](../configure/external-storage/index.md).
+Once DTR has been successfully installed, configure the certificates used for TLS communication ([learn more](../configure/use-your-own-tls-certificates.md)) and the storage backend in which to store the Docker images ([learn more](../configure/external-storage/index.md)).
 
-### Web interface
+### Web Interface
 
-  * To update your TLS certificates, access DTR from the browser and navigate to **System > General**.
-  * To configure your storage backend, navigate to **System > Storage**. If you are upgrading and changing your existing storage backend, see [Switch storage backends](/ee/dtr/admin/configure/external-storage/storage-backend-migration/) for recommended steps.
+  * To update the TLS certificates, access DTR from the browser and navigate to **System > General**.
+  * To configure the storage backend, navigate to **System > Storage**. If you are upgrading and changing the existing storage backend, refer to [Switch storage backends](/ee/dtr/admin/configure/external-storage/storage-backend-migration/).
 
-### Command line interface
+### Command Line Interface
 
-  To reconfigure DTR using the CLI, see the reference page for [the reconfigure command](/reference/dtr/2.7/cli/reconfigure/).
+  To reconfigure DTR using the CLI, refer to the [the reconfigure command](/reference/dtr/2.7/cli/reconfigure/) reference page.
 
-## Step 6. Test pushing and pulling
+## 6. Push and Pull Testing
 
-Now that you have a working installation of DTR, you should test that you can
-push and pull images:
+Once the DTR instance is up and running, test whether images can be pushed and pulled.
 
 * [Configure your local Docker Engine](../../user/access-dtr/index.md)
 * [Create a repository](../../user/manage-images/index.md)
 * [Push and pull images](../../user/manage-images/pull-and-push-images.md)
 
-## Step 7. Join replicas to the cluster
-
-This step is optional.
+## 7. Join Replicas to the Cluster (optional)
 
-To set up DTR for high availability,
-you can add more replicas to your DTR cluster. Adding more replicas allows you
-to load-balance requests across all replicas, and keep DTR working if a
-replica fails.
+To set up DTR for high availability, add more replicas to your DTR cluster. Doing this allows for the load-balancing of requests across all replicas, and it will keep DTR working in the event that a replica fails.
 
-For high-availability, you should set 3 or 5 DTR replicas. The replica nodes also need
-to be managed by the same UCP.
+For high-availability, set 3 or 5 DTR replicas. in addition, t\he replica nodes must be managed by the same UCP.
```

Comment:
"high availability"
"In addition, the..."
"managed by the same UCP instance."

Reply:
Got it. Done. Resolved.